instrumentation/net/http/otelhttp/common.go

```diff
@@ -36,6 +36,11 @@ const (
 	ServerLatency         = "http.server.duration"                // Incoming end to end duration, microseconds
 )
 
+// Client HTTP metrics
+const (
+	ClientRequestDuration = "http.client.duration" // Incoming end to end duration, microseconds
```

Comment:
Is there a reason this is exported? Also, this measures the duration of the outbound HTTP request.

```suggestion
	// clientRequestDuration is the name of the instrument that measures the duration of outbound HTTP requests.
	clientRequestDuration = "http.client.duration"
```

Reply:
I changed it